applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/moment_ref.dat

```diff
@@ -0,0 +1 @@
+-0.105810071870
```

Comment:
Same.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668